common/lib/container-definitions/src/loader.ts

```diff
@@ -136,8 +136,17 @@ export namespace ConnectionState {
     export type Disconnected = 0;
 
     /**
+     * @deprecated - Not yet in use
```

Comment:
I feel like this doesn't really fit how we generally use the deprecated tag. Maybe we can use the experimental tag instead? I don't think it's a perfect fit either but may be less confusing for people who stumble upon this.
https://tsdoc.org/pages/tags/experimental/

Reply:
Yeah it felt weird, just trying it out.  Maybe I don't need to annotate it at all, since the `connectionState` property excludes it explicitly.  And I plan to add an event which can include it in some 1.x release of client packages, but this may still say deprecated.